[2022-04-30 23:39:52,063] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dataproc_bqext_bqpart_dag.create_dataproc_cluster manual__2022-04-30T23:39:40.736273+00:00 [queued]>
[2022-04-30 23:39:52,164] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dataproc_bqext_bqpart_dag.create_dataproc_cluster manual__2022-04-30T23:39:40.736273+00:00 [queued]>
[2022-04-30 23:39:52,170] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-30 23:39:52,172] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-30 23:39:52,174] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-30 23:39:52,255] {taskinstance.py:1259} INFO - Executing <Task(DataprocCreateClusterOperator): create_dataproc_cluster> on 2022-04-30 23:39:40.736273+00:00
[2022-04-30 23:39:52,293] {standard_task_runner.py:52} INFO - Started process 164 to run task
[2022-04-30 23:39:52,335] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dataproc_bqext_bqpart_dag', 'create_dataproc_cluster', 'manual__2022-04-30T23:39:40.736273+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dataproc_spark_dagv3.py', '--cfg-path', '/tmp/tmpwf3hp1i6', '--error-file', '/tmp/tmpgx3i8610']
[2022-04-30 23:39:52,339] {standard_task_runner.py:77} INFO - Job 3: Subtask create_dataproc_cluster
[2022-04-30 23:39:52,534] {logging_mixin.py:109} INFO - Running <TaskInstance: dataproc_bqext_bqpart_dag.create_dataproc_cluster manual__2022-04-30T23:39:40.736273+00:00 [running]> on host 87df04080a08
[2022-04-30 23:39:52,677] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataproc_bqext_bqpart_dag
AIRFLOW_CTX_TASK_ID=create_dataproc_cluster
AIRFLOW_CTX_EXECUTION_DATE=2022-04-30T23:39:40.736273+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-30T23:39:40.736273+00:00
[2022-04-30 23:39:52,680] {dataproc.py:662} INFO - Creating cluster: dtc-capstone-cluster
[2022-04-30 23:39:52,689] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-30 23:43:01,101] {dataproc.py:606} INFO - Cluster created.
[2022-04-30 23:43:01,194] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dataproc_bqext_bqpart_dag, task_id=create_dataproc_cluster, execution_date=20220430T233940, start_date=20220430T233952, end_date=20220430T234301
[2022-04-30 23:43:01,319] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-30 23:43:01,537] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
